script/QA/QA_18.txt

```diff
@@ -51,232 +54,267 @@ And who the heck are you?
 [sha:ce63147b691b36de98a41bd506415b1973d1b2e6]{
 -- Page 117, Offset 43531.
 -- アタシと同類……つまり
-It seems we are the same type... which means
+I can tell you're just like me,
 }
 [sha:dba0e357ea35732683e6d61eea15757c4df4f083]{
 -- Page 117, Offset 43532.
 -- いいところのお嬢さんだっつーのは分かるけど……
-that you must be a young lady with a good background.
+ in that you're a%{n}
+respectable young lady, but...
 }
 [sha:fc6bb37a09829bd452c659d48291d113b28abc02]{
 -- Page 118, Offset 43533.
 -- ほら、こういう同一存在って、
-I mean, species of the same type
+I mean, as two sides of the same existence,%{n}
 }
 [sha:09a79cb6aa14d41d39a9453b18534e4a0ed12df1]{
 -- Page 118, Offset 43534.
 -- お互い認識できないものじゃニャい？
-should be able to recognize one another, right?
+we ought to be able to understand one%{n}
+another, right?
 }
 [sha:c49014af8e9774e10e02981276388b1885cd032a]{
 -- Page 119, Offset 43535.
 -- 自分とは会話できない、みたいな。
-It looks like you're not able to talk to me.
+But isn't there some rule about how you can't%{n}
+meet yourself?
 }
 [sha:fbf10c76f47469b4517313d9973e650fe71f7d91]{
 -- Page 119, Offset 43536.
 -- オルタナティヴな関係ってあるでしょ？
-This must be some kind of alternative relationship, right?
+// Can't find any clear reference on the exact intent of オルタナティヴな関係.
+// No clear patterns to the uses I found in research papers.
+// Feels weird for it to be straight literal here though.
+// My take on this whole bit is that neco arc is saying
+// "lol you're me, that's weird, isn't that not supposed to be possible (世界のルール)"?
+// Not at all confident on this whole exchange.
+ So we must be related some%{n}
+other way.
```

Comment:
Think it's about "alternate universes" or whatever.

>Heard about the relation between alternate universe, right?

Reply:
As in, "You've heard about alternative universes, right?" or "You've heard about the way that alternate universes are related to one another, right?" Not entirely clear from the suggestion.